src/Uno.UI/UI/Xaml/Style/Generic/DatePickerSelector.xaml

```diff
@@ -15,16 +15,48 @@
 		<ios:Setter Property="Template">
 			<Setter.Value>
 				<ControlTemplate TargetType="DatePickerSelector">
-					<Border Background="White"
-							Margin="5"
-							CornerRadius="10">
-						<native:UIDatePicker/>
-					</Border>
+					<Grid VerticalAlignment="Stretch">
+						<Border x:Name="FlyoutScrim"
+								Background="#8000"
+								VerticalAlignment="Stretch"
+								HorizontalAlignment="Stretch"/>
+						<Border x:Name="FlyoutContent"
+								Background="White"
+								VerticalAlignment="Bottom">
+							<StackPanel>
+								<Border Height="44"
+										Background="White">
+									<!--Customize this section with the desired content. Don't forget to localize your resources-->
+									<Button Content="Done"
```

Comment:
Seems like UWP puts the accept and dismiss buttons in the `DatePickerFlyoutPresenter` template. It would be good to match that.

You can check the `DatePickerFlyoutPresenter` style in `C:\Program Files (x86)\Windows Kits\10\DesignTime\CommonConfiguration\Neutral\UAP\10.0.17763.0\Generic\generic.xaml`